<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cmdrundialogClass</class>
 <widget class="QDialog" name="cmdrundialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cmdrundialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:1, y1:0.67, x2:1, y2:0, stop:0.204545 rgba(98, 122, 179, 255), stop:1 rgba(91, 188, 241, 255));

color: rgb(255, 255, 255);
border-style: solid;
border-width: 2px;
border-radius:9px;
}
QGroupBox#groupBox {
	border: 2px solid rgb(61, 139, 255);
	border-radius: 8px;	
	background-color: rgb(221, 221, 221);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{ background-color: rgba(38, 38, 38, 200); }</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>721</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>721</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.67, x2:1, y2:0, stop:0.204545 rgba(98, 122, 179, 255), stop:1 rgba(91, 188, 241, 255));

color: rgb(255, 255, 255);
border-style: solid;
border-width: 2px;
border-radius:9px;
border-color: rgb(61, 139, 255);</string>
     </property>
     <property name="text">
      <string>命令行</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>执行结果</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>551</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(203, 203, 203);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>551</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请输入命令</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_close">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_run">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>320</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>执行</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_stop">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>320</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>pb_close</tabstop>
  <tabstop>pb_run</tabstop>
  <tabstop>pb_stop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
